### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by mccordgh**
# Description

# Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request;
A reference to a related issue in your repository.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation

[ ] There is new documentation in this pull request that must be reviewed..
A description of the changes proposed in the pull request.
@mentions of the person or team responsible for reviewing proposed changes.

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
npm install -g terminal-shark
git checkout <feature_branch>
run terminal-shark
```

## Steps to Test
Outline the steps to test the code in this pull request.

1. 
2.
3.
...

### CODE REVIEW

**Comment by nchemsak** [comment]
👍

**Review by asimonia — APPROVED** [review]
👍

**Review by rtwhitfield84 — APPROVED** [review]
👍

**Comment by danirenae** [comment]
yeehaw! 👍

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-
-## Steps to Test
-Outline the steps to test the code in this pull request.
-
-1. 
-2.
-3.
-...
+Provide a general summary of your changes in the Title above
+
```
Comment (asimonia) [inline PULL_REQUEST_TEMPLATE.md:2]: This looks good 💯

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
+ Please link to the issue here: 
+
+## Motivation and Context
+ Why is this change required? What problem does it solve? 
+
+## How Has This Been Tested?
+ Please describe in detail how you tested your changes. 
+ Include details of your testing environment, and the tests you ran to 
+ see how your change affects other areas of the code, etc. 
+
```
Comment (asimonia) [inline PULL_REQUEST_TEMPLATE.md:19]: I approve of this message.

**Review by asimonia — COMMENTED** [review]
👍

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,34 +1,35 @@
-# Description
-
-# Related Ticket(s)
-List related ticket(s) that apply to the work done in this pull request;
-A reference to a related issue in your repository.
-
```
Comment (asimonia) [inline PULL_REQUEST_TEMPLATE.md:6]: Here is a comment

**Pull request merged**